### ISSUE TITLE
Connecting hand pad 

### ISSUE THREAD

**Issue by AstroKeith** [WimDeMeester/eFinder#52]
I've found that the device name assigned to a USB device isn't fixed. Especially true if more than one is plugged in.
Currently Display.py assumes "/dev/ttyACM1"
The following script will return the actual name assigned.

import subprocess
result = subprocess.check_output('ls -l /dev/serial/by-id/',shell=True)
result =(result.decode('ascii')).splitlines()
for n in range(len(result)):
    if "MicroPython" in result[n]:
        dev_name = "/dev/"+ result[n][-7:]
        print (dev_name)

**Comment by mrosseel** [comment]
My raspbian does not have the /dev/serial/by-id/ folder

**Comment by AstroKeith** [comment]
I see.
I tried it on a fresh Pi with nothing plugged into USB, and you're right.
Then I hot plugged in the hand box. Still no joy.
Then I rebooted the PI, with the handbox plugged in. Yey - it's there!
Unplugged the handbox - it goes.
Rebooted the Pi without handbox - not there.
Hot plugged in the handbox - it appears.

So the safest way, is to boot up the PI with the handbox plugged in. (I find this the most stable anyway)
